domain/domain.go

```diff
@@ -655,6 +655,14 @@ func NewDomain(store kv.Storage, ddlLease time.Duration, statsLease time.Duratio
 
 // Init initializes a domain.
 func (do *Domain) Init(ddlLease time.Duration, sysFactory func(*Domain) (pools.Resource, error)) error {
+	if err := do.InitWithNoRegister(ddlLease, sysFactory); err != nil {
```

Comment:
This also disables the DDL syncer, will it cause problems?

Reply:
Seems `InfoSyncer` won't do things about DDL... Is there something I ignored...?

Reply:
I don't know how BR uses the DDL structure. If BR uses `InitWithNoRegister` to new a DDL, then the DDL also will campaign DDL owner, but it will make load schema slower then `Init` with `InfoSyncer`. Is it OK?